src/dfx-core/src/extension/manager/list.rs

```diff
@@ -0,0 +1,49 @@
+use std::vec;
+
```

Comment:
We tried this, and eventually decided not to break up `use` statements, because various IDEs do different things in that case and it sometimes introduced extra changes.

```suggestion
```

Reply:
Fixed at https://github.com/dfinity/sdk/pull/3896/commits/3791d56c3329f2fb92e9bb701cf296d0ac2cc9bb